addons/op-badge/addon.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "Show OP badge in forums and projects",
+  "description": "Adds a OP badge to the user that created a forum topic or project.",
```

@WorldLanguages Sure thing! How about this?
```suggestion
  "name": "Highlight forum topic/project creator in conversations",
  "description": "Adds a badge next to the original poster in forum topics and next to the project creator in project comments.",
```

---

This one's hard...

---

Should we mention forum topics or projects in the name? I think mentioning both makes it too long.

---

I would say we prioritize project comments and "project creator" and maybe in the future we can consider whether it makes sense to split this addon into two. It might make sense if #6009 and #5937 both go live 🤔

---

```suggestion
  "name": "Highlight project creator in comments",
  "description": "Adds a badge next to the project creator in project comments sections. Also adds a badge next to the original poster in forum topics.",
```